### PR TITLE
RE-Somi

### PR DESCRIPTION
**Pull request by Draxtheros**
New species, except not. This ports Resomi back into the code, with lore in the works for it's final inclusion.

### CODE REVIEW

File: code/modules/species/species.dm [hunk 1]
```diff
@@ -35,6 +35,7 @@
 	var/tail_blend = ICON_ADD
 	var/tail_hair
 
+
```
Comment (PsyCommando) [inline code/modules/species/species.dm]: Would be good to avoid commiting whitespace changes.

**Review by PsyCommando — CHANGES_REQUESTED** [review]
Looks fine! 
Can you just add proper unit tests for the resomi too?

**Comment by Draxtheros** [comment]
Typo in the unit tests fixed, as well as the extra white space

**Review by PsyCommando — APPROVED** [review]

**Comment by PsyCommando** [comment]
Alright, looks good to me then!

**Pull request merged**